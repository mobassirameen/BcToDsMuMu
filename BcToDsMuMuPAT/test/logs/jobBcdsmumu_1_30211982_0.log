000 (30211982.000.000) 12/05 02:02:56 Job submitted from host: <131.225.188.57:9618?addrs=131.225.188.57-9618&noUDP&sock=2564_44eb_3>
...
001 (30211982.000.000) 12/05 02:03:26 Job executing on host: <131.225.206.189:9618?addrs=131.225.206.189-9618&noUDP&sock=1159_8e5a_3>
...
006 (30211982.000.000) 12/05 02:03:34 Image size of job updated: 24
	24  -  MemoryUsage of job (MB)
	25460  -  ResidentSetSize of job (KB)
...
006 (30211982.000.000) 12/05 02:04:34 Image size of job updated: 197
	197  -  MemoryUsage of job (MB)
	202024  -  ResidentSetSize of job (KB)
...
006 (30211982.000.000) 12/05 02:05:34 Image size of job updated: 207
	207  -  MemoryUsage of job (MB)
	211988  -  ResidentSetSize of job (KB)
...
006 (30211982.000.000) 12/05 02:06:34 Image size of job updated: 365
	365  -  MemoryUsage of job (MB)
	373808  -  ResidentSetSize of job (KB)
...
006 (30211982.000.000) 12/05 02:08:34 Image size of job updated: 597
	597  -  MemoryUsage of job (MB)
	612072  -  ResidentSetSize of job (KB)
...
006 (30211982.000.000) 12/05 02:10:36 Image size of job updated: 597
	597  -  MemoryUsage of job (MB)
	612096  -  ResidentSetSize of job (KB)
...
006 (30211982.000.000) 12/05 02:11:37 Image size of job updated: 597
	597  -  MemoryUsage of job (MB)
	612100  -  ResidentSetSize of job (KB)
...
006 (30211982.000.000) 12/05 02:15:37 Image size of job updated: 621
	621  -  MemoryUsage of job (MB)
	636300  -  ResidentSetSize of job (KB)
...
006 (30211982.000.000) 12/05 02:16:37 Image size of job updated: 621
	621  -  MemoryUsage of job (MB)
	636352  -  ResidentSetSize of job (KB)
...
006 (30211982.000.000) 12/05 02:18:39 Image size of job updated: 621
	621  -  MemoryUsage of job (MB)
	636584  -  ResidentSetSize of job (KB)
...
005 (30211982.000.000) 12/05 02:23:35 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:16:06, Sys 0 00:00:13  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:16:06, Sys 0 00:00:13  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	47070  -  Run Bytes Sent By Job
	277352  -  Run Bytes Received By Job
	47070  -  Total Bytes Sent By Job
	277352  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :        0        1         1 
	   Disk (KB)            :      338    15000    333918 
	   Memory (MB)          :      621     8000      8000 
...
