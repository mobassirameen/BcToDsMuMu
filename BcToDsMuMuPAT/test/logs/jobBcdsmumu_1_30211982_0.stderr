./condor_exec.exe: line 35: limit: command not found
which: no edg-gridftp-ls in (/cvmfs/cms.cern.ch/common:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
which: no srmcp in (/cvmfs/cms.cern.ch/common:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
/storage/local/data1/condor/execute/dir_24006/CMSSW_10_2_18/src/bctodsmumu_analysis/BcToDsMuMuPAT/src/BcToDsMuMu.cc: In member function 'virtual void BcToDsMuMu::analyze(const edm::Event&, const edm::EventSetup&)':
/storage/local/data1/condor/execute/dir_24006/CMSSW_10_2_18/src/bctodsmumu_analysis/BcToDsMuMuPAT/src/BcToDsMuMu.cc:666:9: warning: unused variable 'Ds_sigma' [-Wunused-variable]
   float Ds_sigma = Ds_mass*1.e-6;
         ^~~~~~~~
/storage/local/data1/condor/execute/dir_24006/CMSSW_10_2_18/src/bctodsmumu_analysis/BcToDsMuMuPAT/src/BcToDsMuMu.cc:534:17: warning: unused variable 'psi_mass' [-Wunused-variable]
    ParticleMass psi_mass = 3.096916;
                 ^~~~~~~~
/storage/local/data1/condor/execute/dir_24006/CMSSW_10_2_18/src/bctodsmumu_analysis/BcToDsMuMuPAT/src/BcToDsMuMu.cc: In member function 'bool BcToDsMuMu::IsTheSametk(const pat::GenericParticle&, const pat::Muon&)':
/storage/local/data1/condor/execute/dir_24006/CMSSW_10_2_18/src/bctodsmumu_analysis/BcToDsMuMuPAT/src/BcToDsMuMu.cc:1105:10: warning: unused variable 'DeltaEta' [-Wunused-variable]
   double DeltaEta = fabs(mu.eta()-tk.eta());
          ^~~~~~~~
/storage/local/data1/condor/execute/dir_24006/CMSSW_10_2_18/src/bctodsmumu_analysis/BcToDsMuMuPAT/src/BcToDsMuMu.cc:1106:10: warning: unused variable 'DeltaP' [-Wunused-variable]
   double DeltaP   = fabs(mu.p()-tk.p());
          ^~~~~~
/storage/local/data1/condor/execute/dir_24006/CMSSW_10_2_18/src/bctodsmumu_analysis/BcToDsMuMuPAT/src/BcToDsMuMu.cc: In member function 'bool BcToDsMuMu::IsTheSamePFtk(const pat::PackedCandidate&, const pat::Muon&)':
/storage/local/data1/condor/execute/dir_24006/CMSSW_10_2_18/src/bctodsmumu_analysis/BcToDsMuMuPAT/src/BcToDsMuMu.cc:1112:10: warning: unused variable 'DeltaEta' [-Wunused-variable]
   double DeltaEta = fabs(mu.eta()-tk.eta());
          ^~~~~~~~
/storage/local/data1/condor/execute/dir_24006/CMSSW_10_2_18/src/bctodsmumu_analysis/BcToDsMuMuPAT/src/BcToDsMuMu.cc:1113:10: warning: unused variable 'DeltaP' [-Wunused-variable]
   double DeltaP   = fabs(mu.p()-tk.p());
          ^~~~~~
cp: missing destination file operand after '../../../../../BcToDsMuMuRootupler_mod.py'
Try 'cp --help' for more information.
05-Dec-2020 08:05:59 UTC  Initiating request to open file root://cmsxrootd-site.fnal.gov//store/data/Run2018B/DoubleMuonLowMass/MINIAOD/17Sep2018-v1/100000/32AB25E3-8BF6-ED4F-A72B-FF5E7036C416.root
%MSG-w XrdAdaptor:  file_open 05-Dec-2020 08:06:00 UTC pre-events
Data is served from fnal.gov instead of original site T1_US_FNAL
%MSG
05-Dec-2020 08:06:02 UTC  Successfully opened file root://cmsxrootd-site.fnal.gov//store/data/Run2018B/DoubleMuonLowMass/MINIAOD/17Sep2018-v1/100000/32AB25E3-8BF6-ED4F-A72B-FF5E7036C416.root
Begin processing the 1st record. Run 317649, Event 353296335, LumiSection 239 on stream 0 at 05-Dec-2020 08:07:24.389 UTC
Begin processing the 2nd record. Run 317649, Event 353124380, LumiSection 239 on stream 0 at 05-Dec-2020 08:07:46.027 UTC
Begin processing the 3rd record. Run 317649, Event 352908303, LumiSection 239 on stream 0 at 05-Dec-2020 08:08:14.732 UTC
Begin processing the 4th record. Run 317649, Event 353294279, LumiSection 239 on stream 0 at 05-Dec-2020 08:08:14.739 UTC
Begin processing the 5th record. Run 317649, Event 353312715, LumiSection 239 on stream 0 at 05-Dec-2020 08:08:24.581 UTC
Begin processing the 6th record. Run 317649, Event 353313689, LumiSection 239 on stream 0 at 05-Dec-2020 08:08:41.305 UTC
Begin processing the 7th record. Run 317649, Event 352896614, LumiSection 239 on stream 0 at 05-Dec-2020 08:08:41.314 UTC
Begin processing the 8th record. Run 317649, Event 352934049, LumiSection 239 on stream 0 at 05-Dec-2020 08:09:57.412 UTC
Begin processing the 9th record. Run 317649, Event 353250971, LumiSection 239 on stream 0 at 05-Dec-2020 08:09:57.426 UTC
Begin processing the 10th record. Run 317649, Event 353163601, LumiSection 239 on stream 0 at 05-Dec-2020 08:09:57.434 UTC
Begin processing the 11th record. Run 317649, Event 353094110, LumiSection 239 on stream 0 at 05-Dec-2020 08:09:57.439 UTC
Begin processing the 12th record. Run 317649, Event 353024448, LumiSection 239 on stream 0 at 05-Dec-2020 08:12:32.698 UTC
Begin processing the 13th record. Run 317649, Event 353019981, LumiSection 239 on stream 0 at 05-Dec-2020 08:12:32.729 UTC
Begin processing the 14th record. Run 317649, Event 353346060, LumiSection 239 on stream 0 at 05-Dec-2020 08:13:04.722 UTC
Begin processing the 15th record. Run 317649, Event 353170133, LumiSection 239 on stream 0 at 05-Dec-2020 08:13:11.067 UTC
Begin processing the 16th record. Run 317649, Event 352916205, LumiSection 239 on stream 0 at 05-Dec-2020 08:13:21.899 UTC
Begin processing the 17th record. Run 317649, Event 353144607, LumiSection 239 on stream 0 at 05-Dec-2020 08:13:21.918 UTC
Begin processing the 18th record. Run 317649, Event 353106918, LumiSection 239 on stream 0 at 05-Dec-2020 08:13:29.395 UTC
Begin processing the 19th record. Run 317649, Event 352984396, LumiSection 239 on stream 0 at 05-Dec-2020 08:13:29.402 UTC
Begin processing the 20th record. Run 317649, Event 353061453, LumiSection 239 on stream 0 at 05-Dec-2020 08:13:59.002 UTC
Begin processing the 21st record. Run 317649, Event 353792089, LumiSection 239 on stream 0 at 05-Dec-2020 08:14:54.916 UTC
Begin processing the 22nd record. Run 317649, Event 352813316, LumiSection 239 on stream 0 at 05-Dec-2020 08:15:04.153 UTC
Begin processing the 23rd record. Run 317649, Event 353271872, LumiSection 239 on stream 0 at 05-Dec-2020 08:15:15.661 UTC
Begin processing the 24th record. Run 317649, Event 353336641, LumiSection 239 on stream 0 at 05-Dec-2020 08:15:15.675 UTC
Begin processing the 25th record. Run 317649, Event 353018070, LumiSection 239 on stream 0 at 05-Dec-2020 08:16:26.109 UTC
Begin processing the 26th record. Run 317649, Event 353249139, LumiSection 239 on stream 0 at 05-Dec-2020 08:16:39.224 UTC
Begin processing the 27th record. Run 317649, Event 353055732, LumiSection 239 on stream 0 at 05-Dec-2020 08:16:51.816 UTC
Begin processing the 28th record. Run 317649, Event 353225121, LumiSection 239 on stream 0 at 05-Dec-2020 08:17:39.968 UTC
Begin processing the 29th record. Run 317649, Event 352912778, LumiSection 239 on stream 0 at 05-Dec-2020 08:18:31.463 UTC
Begin processing the 30th record. Run 317649, Event 352775376, LumiSection 239 on stream 0 at 05-Dec-2020 08:18:56.961 UTC
Begin processing the 31st record. Run 317649, Event 352835640, LumiSection 239 on stream 0 at 05-Dec-2020 08:19:57.769 UTC
Begin processing the 32nd record. Run 317649, Event 352784215, LumiSection 239 on stream 0 at 05-Dec-2020 08:20:21.926 UTC
Begin processing the 33rd record. Run 317649, Event 353029728, LumiSection 239 on stream 0 at 05-Dec-2020 08:20:46.490 UTC
Begin processing the 34th record. Run 317649, Event 353253727, LumiSection 239 on stream 0 at 05-Dec-2020 08:22:35.651 UTC
Begin processing the 35th record. Run 317649, Event 353078361, LumiSection 239 on stream 0 at 05-Dec-2020 08:22:35.661 UTC
Begin processing the 36th record. Run 317649, Event 352730101, LumiSection 239 on stream 0 at 05-Dec-2020 08:22:43.055 UTC


A fatal system signal has occurred: segmentation violation
The following is the call stack containing the origin of the signal.


Current Modules:

Module: BcToDsMuMu:rootuple (crashed)

A fatal system signal has occurred: segmentation violation
./condor_exec.exe: line 77:   968 Segmentation fault      (core dumped) cmsRun BcToDsMuMuRootupler_mod.py
[387B/387B][100%][==================================================][387B/s]  [387B/387B][100%][==================================================][387B/s]  
